content/technotes/2018-04-27-nomisr-access-uk-census-data.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "Nomisr - Access 'Nomis' UK Labour Market Data"
```

Comment:
Please add slug and topicid to YAML e.g.
`slug: "nomisr"`
`topicid: `

I will add the topicid at the time of publication.

Reply:
Please knit the .Rmd to .md for rendering on our site.